packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -314,6 +313,9 @@ const AccountWizard = observer(props => {
             delete clone.tax_identification_number;
         }
 
+        if (clone?.tnc_acceptance[0] === 'on') {
```

Comment:
why  `tnc_acceptance` is an array? isnt it boolean value?

Reply:
When i consoled and check the value for clone.tnc_acceptance returned was ["on"]. I believe since its a checkbox it returns "on" inside the array.

Reply:
what is expected for BE to be sent?

Reply:
sorry, ignore it. I see :)

Reply:
what is the value for clone.tnc_acceptance if it is not clicked?

Reply:
@utkarsha-deriv can you please check the `payload` instead of clone.tnc_acceptance

Reply:
I checked again and it returns me `true`, Im not sure why it was returning ["on"] previously 🤔 . Since, i tested it also and the condition to check for `clone?.tnc_acceptance`  was becoming true. We can also see tnc_acceptance: 1 in request. 
![Screenshot 2024-08-15 at 6 15 09 PM](https://github.com/user-attachments/assets/fd1acc78-dfc2-425c-a3cd-25f0447ac963)